docs/reference/troubleshooting/common-issues/rejected-requests.asciidoc

```diff
@@ -32,6 +32,9 @@ correlating log as `EsRejectedExecutionException` with either
 
 These errors are often related to <<task-queue-backlog,backlogged tasks>>.
 
+See https://www.youtube.com/watch?v=auZJRXoAVpI[this video] for an example
```

Comment:
can you adjust these sentences so they match the style of the [sentence in the circuit breaker doc](https://github.com/elastic/elasticsearch/pull/119311/files#diff-8fbda56e6a0463550ff8f4e00ba7902df7e7a7d62b5fcf9f21f51d5cdff3325eR12)?

Reply:
Done 💚